src/lando/dockerflow/views.py

```diff
@@ -0,0 +1,83 @@
+from django.db import connections
+from django.db.utils import OperationalError
+from django.http import JsonResponse
+from django.views import View
+
+from lando.dockerflow.decorators import disable_caching, log_request
+
+
+class VersionView(View):
+    """
+    This view handles the version information of Lando.
+
+    It returns a JSON response containing the version information.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    def get(self, request):
+        try:
+            from lando.version import version
+
+            data = {
+                "version": version,
+            }
+            status_code = 200
+        except ImportError:
+            data = {"error": "Service Unavailable"}
+            status_code = 503
+
+        return JsonResponse(data, status=status_code, json_dumps_params={"indent": 2})
```

Comment:
`json_dumps_params={"indent": 2}` is repeated in all these views. The value at least, can probably go in the parent class. Could also just define a `self._json_response(data, status)` that automatically fills in the necessary values.

E.g., 

```python
return self._json_response(data, status)
```

and elsewhere

```python
@staticmethod
def _json_response(self, data, status):
    return JsonResponse(data, status=status, json_dump_params={"indent": 2})

Reply:
Done.